content/en/security/notifications/variables.md

```diff
@@ -22,17 +22,21 @@ Use template variables to inject dynamic context from triggered logs or traces d
-| `{{severity}}`        | The severity of the triggering rule case (integer, 0-4).                                      |
-| `{{timestamp}}`       | Time the signal was created. For example, `Mon Jan 01 00:00:00 UTC 1970`.                     |
-| `{{timestamp_epoch}}` | Time the signal was created, in milliseconds since midnight, January 1, 1970.                 |
-| `{{first_seen}}`      | Time the signal was first seen. For example, `Mon Jan 01 00:00:00 UTC 1970`.                  |
-| `{{first_seen_epoch}}`| Time the signal was first seen, in milliseconds since midnight, January 1, 1970.              |
-| `{{last_seen}}`       | Time the signal was most recently triggered. For example, `Mon Jan 01 00:00:00 UTC 1970`.     |
-| `{{last_seen_epoch}}` | Time the signal was most recently triggered, in milliseconds, since midnight, January 1, 1970.|
-| `{{rule_name}}`       | Name of the associated rule.                                                                  |
-| `{{case_name}}`       | Name of the triggering rule case.                                                             |
+| Variable                                           | Description                                                                                   |
+| -------------------------------------------------- | --------------------------------------------------------------------------------------------- |
+| `{{severity}}`                                     | The severity of the triggering rule case (integer, 0-4).                                      |
+| `{{timestamp}}`                                    | Time the signal was created. For example, `Mon Jan 01 00:00:00 UTC 1970`.                     |
+| `{{timestamp_epoch}}`                              | Time the signal was created, in milliseconds since midnight, January 1, 1970.                 |
+| `{{first_seen}}`                                   | Time the signal was first seen. For example, `Mon Jan 01 00:00:00 UTC 1970`.                  |
+| `{{first_seen_epoch}}`                             | Time the signal was first seen, in milliseconds since midnight, January 1, 1970.              |
+| `{{last_seen}}`                                    | Time the signal was most recently triggered. For example, `Mon Jan 01 00:00:00 UTC 1970`.     |
+| `{{last_seen_epoch}}`                              | Time the signal was most recently triggered, in milliseconds, since midnight, January 1, 1970.|
+| `{{rule_name}}`                                    | Name of the associated rule.                                                                  |
+| `{{case_name}}`                                    | Name of the triggering rule case.                                                             |
+| `{{events_matched}}`                               | Number of events which has matched the associated rule.                                       |
```

Comment:
```suggestion
| `{{events_matched}}`                               | Number of events that have matched the associated rule.                                       |
```

Reply:
Done in https://github.com/DataDog/documentation/pull/22053/commits/567486cdbb6ca900f898eb788d9885a54e4a98ff